packages/select-rich/test/lion-select-rich.test.js

```diff
@@ -0,0 +1,193 @@
+  });
+
+  it('can preselect an option', async () => {
+    const el = await fixture(html`
+      <lion-select-rich .modelValue="${'nr2'}">
+        <lion-listbox slot="input">
+          <lion-option value="nr1">Item 1</lion-option>
+          <lion-option value="nr2">Item 2</lion-option>
+        </lion-listbox>
+      </lion-select-rich>
+    `);
+    expect(el._listboxElement.value).to.equal('nr2');
+  });
+
+  it('shows content of the selected option inside the invoker', async () => {
+    const el = await fixture(html`
+      <lion-select-rich>
+        <lion-listbox slot="input">
+          <lion-option value="nr1" selected><span>Item 1</span></lion-option>
+        </lion-listbox>
+      </lion-select-rich>
+    `);
+    expect(el._invokerElement.innerHTML).to.equal('<span>Item 1</span>');
+  });
+
+  it('has a class "state-disabled"', async () => {});
+
+  describe('Interaction states', () => {});
+
+  describe('Validation', () => {
+    it('can be required', async () => {
+      const lionSelect = await fixture(html`
+        <lion-select-rich name="foo" .errorValidators="${[`required`]}">
+          <lion-listbox slot="input">
+            <lion-option value="nr1">Item 1</lion-option>
+            <lion-option value="nr2">Item 2</lion-option>
+          </lion-listbox>
+        </lion-select-rich>
+      `);
+      expect(lionSelect.error.required).to.be.true;
+      lionSelect.modelValue = 'nr2';
+      expect(lionSelect.error.required).to.be.undefined;
+    });
+  });
+
+  describe('Invoker', () => {
+    it('should be shown closed by default', async () => {
+      const el = await fixture(html`
+        <lion-select-rich></lion-select-rich>
+      `);
+      expect(el._listboxElement.style.display).to.be.equal('none');
+    });
+
+    it('can toggle the listbox', async () => {
+      const el = await fixture(html`
+        <lion-select-rich></lion-select-rich>
+      `);
+      expect(el._listboxElement.style.display).to.be.equal('none');
+      el.click();
+      expect(el._listboxElement.style.display).to.be.equal('inline-block');
+    });
+
+    describe('Keyboard navigation', () => {
+      it('opens the listbox with [enter] key', async () => {
+        const el = await fixture(html`
+          <lion-select-rich></lion-select-rich>
+        `);
+        el.dispatchEvent(new KeyboardEvent('keydown', { key: 'Enter' }));
+        await el.updateComplete;
+        expect(el._listboxElement.style.display).to.be.equal('inline-block');
+      });
+
+      it('opens the listbox with [space] key', async () => {
+        const el = await fixture(html`
+          <lion-select-rich></lion-select-rich>
+        `);
+        el.dispatchEvent(new KeyboardEvent('keydown', { key: 'Space' }));
+        await el.updateComplete;
+        expect(el._listboxElement.style.display).to.be.equal('inline-block');
+      });
+
+      it('opens the listbox with [arrow down] key and changes focus to listbox', async () => {
```

Comment:
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

Here, the focus goes to the second option when doing arrow-down (as the first option is preselected by default). Enter / space puts the focus on the first option.

Do we want similar behavior? Namely, having the first option pre-selected by default? And then I guess listbox should not be focusable.

Reply:
Makes sense I guess. Maybe add an extra it for this?